cli/command/system/info.go

```diff
@@ -261,7 +262,19 @@ func printSwarmInfo(dockerCli command.Cli, info types.Info) {
 		fmt.Fprintln(dockerCli.Out(), " ClusterID:", info.Swarm.Cluster.ID)
 		fmt.Fprintln(dockerCli.Out(), " Managers:", info.Swarm.Managers)
 		fmt.Fprintln(dockerCli.Out(), " Nodes:", info.Swarm.Nodes)
+		var strAddrPool strings.Builder
+		if info.Swarm.Cluster.DefaultAddrPool != nil {
+			for _, p := range info.Swarm.Cluster.DefaultAddrPool {
+				strAddrPool.WriteString(p + "  ")
+			}
+			fmt.Fprintln(dockerCli.Out(), " Default Address Pool:", strAddrPool.String())
```

Comment:
oh, sorry, missed that: this should be `" Default Address Pools:"` (plural) here; because we're showing all address pools

Reply:
we had debate on that a while back. pool itself means group of address. so we decided to keep it like that.  I can change it if you want .